test/Common/DurableClientBaseTests.cs

```diff
@@ -335,7 +335,7 @@ private static DurableTaskExtension GetDurableTaskConfig()
                 wrappedOptions,
                 new LoggerFactory(),
                 nameResolver,
-                serviceFactory,
+                new[] { serviceFactory }.ToList(),
```

Comment:
No need for `.ToList()` here.
```suggestion
                new[] { serviceFactory },
```